iModelCore/Units/tests/NonPublished/FormattingTests.cpp

```diff
@@ -920,42 +930,54 @@ TEST_F(FormatParsingSetTest, CompositeFormats_IGNORED)
 TEST_F(FormatParsingSetTest, OverflowNumberTest)
     {
     TestValidParseToQuantity("812345678910111 m", meter, 812345678910111);
-    FormatParsingSet fps("8123456789101110 m", meter);
+    Format format;
+    FormatParsingSet fps("8123456789101110 m", meter, &format);
+    
+    fps.GetQuantity(nullptr, &format);
     EXPECT_TRUE(fps.HasProblem());
     EXPECT_TRUE(fps.GetProblemCode() == FormatProblemCode::TooManyDigits);
 
     TestValidParseToQuantity("0.012345678910111 m", meter, 0.01234567891011101);
     fps = FormatParsingSet("0.0123456789101112 m", meter);
+    fps.GetQuantity(nullptr, &format);
     EXPECT_TRUE(fps.HasProblem()) << "Should be overflow because the number of digits in the floating part is more than 15";
     EXPECT_TRUE(fps.GetProblemCode() == FormatProblemCode::TooManyDigits);
 
     TestValidParseToQuantity("0.000000000000001 m", meter, 0.000000000000001);
-    fps = FormatParsingSet("0.0000000000000001 m", meter);
+    // fps = FormatParsingSet("0.000000000000001 m", meter);
+    // fps.GetQuantity(nullptr, &format);
+    FormatParsingSet fps2("0.000000000000001 m", meter);
+    fps2.GetQuantity(nullptr, &format);

```

Comment:
what's the deal here.  you create fps2 but don't validate it

Reply:
just fixed it